bluesky/examples.py

```diff
@@ -115,6 +115,9 @@ def settle(self):
     def stop(self):
         pass
 
+    def trigger(self):
```

Comment:
As I remember it, we allow for the possibility that users might want to set a motor and then trigger that setting as a separate step. Imagine the case where set performs some moderately expensive computation.